tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -2911,4 +2962,44 @@ public function forUserSpaceShouldContainLinks(
 			Assert::assertEmpty($responseArray, __METHOD__ . ' Response should be empty');
 		}
 	}
+	   
+	/**
+	 * @When /^user "([^"]*)" gets the following properties of (?:file|folder|entry|resource) "([^"]*)" inside space "([^"]*)" using the WebDAV API$/
+	 *
+	 * @param string    $user
+	 * @param string    $resourceName
+	 * @param string    $spaceName
+	 * @param TableNode|null $propertiesTable
+	 *
+	 * @return void
+	 *
+	 * @throws Exception|GuzzleException
+	 */
+	public function userGetsTheFollowingPropertiesOfFileInsideSpaceUsingTheWebdavApi($user, $resourceName, $spaceName, TableNode $propertiesTable):void {
+		$space = $this->getSpaceByName($user, $spaceName);
+		$properties = null;
+		$fullUrl = $this->baseUrl . $this->davSpacesUrl . $space["id"] . $resourceName;
```

Comment:
```suggestion
		$fullUrl = $space["root"]["webDavUrl] . "/" . $resourceName;
```
the WebDAV URL is already available inside the `$space` var. we can use that.

Reply:
Also trim can be used `\ltrim($resourceName, "/");`